benchmark/websocket/simple.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const common = require('../common.js');
+const crypto = require('crypto');
+const http = require('http');
+const { WebSocket } = require('undici');
+
+const GUID = '258EAFA5-E914-47DA-95CA-C5AB0DC85B11';
+
+const configs = {
+  size: [64, 16 * 1024, 128 * 1024, 1024 * 1024],
+  useBinary: ['true', 'false'],
+  roundtrips: [5000, 1000, 100, 1],
```

Comment:
```suggestion
  roundtrips: [5000, 1000, 100],
```

I think a single roundtrip does not make much sense.